packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -184,8 +184,6 @@ export const toggleTor =
         const ipcResponse = await desktopApi.toggleTor(shouldEnable);
 
         if (ipcResponse.success) {
-            window.fetch = shouldEnable ? torFetch : baseFetch;
-
```

Comment:
Why are we removing this? I think even if we use clearnet over Tor we still need this in order to use Tor in the renderer.

Reply:
Dunno, it seems that the only think it does is translating clearnet addresses.
https://github.com/trezor/trezor-suite/blob/861d629ca201ca305a12bfff33fed770f16fa3a8/packages/suite/src/utils/suite/tor.ts#L43-L49

Reply:
Fixup is on the way :+1:

Reply:
What about https://github.com/trezor/trezor-suite/blob/23f18da2fbf12edd0e7daaaf91cd668dfd1cafd7/packages/suite/src/utils/suite/tor.ts#L25 and its tests https://github.com/trezor/trezor-suite/blob/23f18da2fbf12edd0e7daaaf91cd668dfd1cafd7/packages/suite/src/utils/suite/__tests__/tor.test.ts#L106  ? they seam to not be used anymore.

Reply:
You're right. But this is just a common util with good chance of being used somewhere in the future, so I'm not sure... What do you think? Should we remove it?

Reply:
I would remove it. We can always put it back if we need it in the future but having dead code is confusing.
But it is up to you, I get your point.

Reply:
Ok, agreed. https://github.com/trezor/trezor-suite/pull/7117/commits/d55c2191639f6ae945f05f258238bd17eae677d7